docker/build_standalone-tcks.sh

```diff
@@ -27,7 +27,6 @@ if [ -z "$JAKARTA_JARS" ]; then
   export JAKARTA_JARS=$BASEDIR
 fi
 
-export JAVA_HOME=${JDK11_HOME}
```

Comment:
I think that we should always build with JDK11 for EE 10. With this pull request, we should only be changing the running of TCK tests to be able to use either JDK11 or JDK17.

Reply:
With update to docker image "jakartaee/cts-base:0.3" default JAVA_HOME is set to "/opt/jdk-11.0.2"(replacing jdk8 java_home), hence we need not set JAVA_HOME at build script(have removed redundant JAVA_HOME export)